rig/machine_control/machine_controller.py

```diff
@@ -834,9 +904,14 @@ def send_signal(self, signal, app_id=Required):
     def load_routing_tables(self, routing_tables, app_id=Required):
         """Allocate space for an load multicast routing tables.
 
+        The routing table entries will be removed automatically when the
+        associated application is stopped.
```

Comment:
This doesn't appear to happen and I haven't chased down why.  It should though...